googlesearch/__init__.py

```diff
@@ -120,7 +122,7 @@
 
 
 # Get a random user agent.
-def get_random_user_agent():
+def get_random_user_agent() -> str:
```

Comment:
Won't the type hints break compatibility with Python 2.x? I'd like to keep it compatible